templates/landscape/managed.html

```diff
@@ -1,6 +1,6 @@
 {% extends "landscape/base_landscape.html" %}
 
-{% block title %}Install Landscape{% endblock %}
+{% block title %} Managed Landscape {% endblock %}
```

Comment:
is there a reason why you added an extra space before and after?

Reply:
Nah, just habit, I was trying to get this up quickly. For the copydoc block I do it intentionally so that the link is clickable